src/util/orders.test.ts

```diff
@@ -77,3 +77,37 @@ describe('buildOrder', () => {
         );
     });
 });
+describe('orderDetails', () => {
+    it('orderDetailsFeeEther should calculate totalFee ', () => {
+        // given
+        const makerAmount = new BigNumber(5000000000000000000);
+        const takerAmount = new BigNumber(500000000000000000);
+        const orderType = OrderSide.Buy;
+
+        // when
+        const orderInEther = orderDetailsFeeEther(makerAmount, takerAmount, orderType);
```

Comment:
It seems like this function (and therefore these tests) has a implicit dependency on the environment variables.

This is what I would do: make the function accept the maker/taker fee values, but use the environment variables as default arguments. That way you don't have to change the code that uses the function but you can be more explicit in the tests.

Reply:
Also, please add at least one test for a buy order and another for a sell order.